construct.yaml

```diff
@@ -53,7 +51,7 @@ specs:
   - opensearch-dsl
   # FIXME: We need to pin MySQL as 8.0.28 dropped support for TLS v1.0 and v1.1
   # In principle MySQL v5.7.10 supports TLSv1.2 but it wasn't enabled in LHCb at least
-  - mysql-client =8.0.27
```

Comment:
If you're running a databse older than MySQL v5.7.10+, MariaDB 5.5.41+ and MariaDB 10.0.15+ and using TLS this change will break. These releases are very old so hopefully everyone is updated 😉